dev-java/protobuf-java/protobuf-java-9999.ebuild

```diff
@@ -3,52 +3,88 @@
 
 EAPI=8
 
+# Tests not enabled, depend on com.google.truth which is not packaged
+# https://github.com/protocolbuffers/protobuf/blob/v21.5/java/core/pom.xml#L35-L40
 JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.google.protobuf:${PN}:${PV}"
```

Comment:
Putting ${PV} here would resolve to "9999", right?
Carrect would be the value from the https://github.com/protocolbuffers/protobuf/blob/main/java/core/pom.xml#L7

Reply:
Updated.

I think you are right, however it would require to update when new release which defeats the purpose of 9999 ebuilds, but it’s same issue with SLOT, so I considered it okay.

Maybe we can find a way to parse the value thanks to some git commands?